packages/percy-storybook/src/selectStories.js

```diff
@@ -50,6 +59,7 @@ export default function selectStories(rawStories, rtlRegex) {
       assertWidths(options.widths);
       assertRtl(options.rtl);
       assertSkip(options.skip);
+      assertMinimumHeight(options.minimum_height);
```

Comment:
We can bikeshed over snake_case vs camelCase if you want. This currently matches the CLI flag, but I kinda feel like it should match what we have going on for Percy Agent (`minimumHeight` I think?)

Reply:
ha, yes! `minHeight` :-)